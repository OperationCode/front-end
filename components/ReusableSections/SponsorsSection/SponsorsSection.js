import React from 'react';
import Container from 'components/Container/Container';
import Heading from 'components/Heading/Heading';
import PartnerLogoLink from 'components/PartnerLogoLink/PartnerLogoLink';
import partners, { PARTNER_TYPES } from 'common/constants/partners';
import styles from './SponsorsSection.css';

const isPaidSponsor = partner => partner.type === PARTNER_TYPES.PAID;

const SponsorsSection = () => (
  <Container theme="gray">
    <Heading text="Sponsors" hasTitleUnderline headingLevel={3} />

    <Heading text="Paid Sponsors" headingLevel={4} />
    <p>
      These sponsors have donated cash to help keep the mission going here at Operation Code. They
      may or may not have also donated to our cause via others means.
    </p>
    <div className={styles.columnsContainer}>
      {partners
        .filter(x => isPaidSponsor(x))
        .map(partner => (
          <PartnerLogoLink key={partner.name} {...partner} size="large" />
        ))}
    </div>

    <Heading text="In-Kind Sponsors" headingLevel={4} />
    <p>
      The following organizations have helped Operation Code through services, products,
      advertisements, scholarships, or sponsorships. We thank them for their contributions.
    </p>
    <div className={styles.columnsContainer}>
      {partners
        .filter(x => !isPaidSponsor(x))
        .map(partner => (
          <PartnerLogoLink key={partner.name} {...partner} />
        ))}
    </div>
  </Container>
);

export default SponsorsSection;
