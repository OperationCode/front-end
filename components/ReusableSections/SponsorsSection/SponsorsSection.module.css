.justifyAlign {
  text-align: justify;
}

.verticalSpacing {
  margin-top: 1rem;
  margin-bottom: 1rem;
}

.container {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  margin: 1rem auto;
}

.columnsContainer {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-wrap: wrap;
}

/* the individual columns */
.columnsContainer > * {
  margin: 1rem;
}
