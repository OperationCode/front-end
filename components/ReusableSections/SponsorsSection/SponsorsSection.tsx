import { Container } from 'components/Container/Container';
import { Heading } from 'components/Heading/Heading';
import { PartnerLogoLink } from 'components/PartnerLogoLink/PartnerLogoLink';
import { PARTNER_TYPES, sortedPartners } from 'common/constants/partners';

interface Partner {
  name: string;
  type: string;
}

const isPaidSponsor = (partner: Partner): boolean => partner.type === PARTNER_TYPES.PAID;

export const SponsorsSection = () => (
  <Container theme="gray" data-testid="Sponsors Section">
    <Heading text="Sponsors" hasTitleUnderline headingLevel={3} />

    <Heading text="Corporate Partners" headingLevel={4} />
    <p>
      These sponsors have donated cash to help keep the mission going here at Operation Code. They
      may or may not have also donated to our cause via others means.
    </p>
    <div className="flex justify-center items-center flex-wrap [&>*]:m-4">
      {sortedPartners
        .filter(x => isPaidSponsor(x))
        .map(partner => (
          <PartnerLogoLink key={partner.name} {...partner} size="large" />
        ))}
    </div>

    <Heading text="Corporate Supporters" headingLevel={4} />
    <p>
      The following organizations have helped Operation Code through services, products,
      advertisements, scholarships, or sponsorships. We thank them for their contributions.
    </p>
    <div className="flex justify-center items-center flex-wrap [&>*]:m-4">
      {sortedPartners
        .filter(x => !isPaidSponsor(x))
        .map(partner => (
          <PartnerLogoLink key={partner.name} {...partner} />
        ))}
    </div>
  </Container>
);
