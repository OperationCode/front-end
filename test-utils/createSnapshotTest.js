/* eslint-env jest */
import renderer from 'react-test-renderer';

/**
 * Used to create fully-rendered snapshot test.
 * Useful for reusable component smoke tests.
 *
 * @export
 * @param {*} Component
 */
export default Component => {
  const tree = renderer.create(Component).toJSON();
  expect(tree).toMatchSnapshot();
};
