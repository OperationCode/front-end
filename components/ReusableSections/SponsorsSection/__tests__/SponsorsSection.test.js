import React from 'react';
import createSnapshotTest from 'test-utils/createSnapshotTest';

import SponsorsSection from '../SponsorsSection';

describe('SponsorsSection', () => {
  it('should render with required props', () => {
    createSnapshotTest(<SponsorsSection>Test</SponsorsSection>);
  });

  it('should render with many props assigned', () => {
    createSnapshotTest(<SponsorsSection className="test-class">Test</SponsorsSection>);
  });
});
