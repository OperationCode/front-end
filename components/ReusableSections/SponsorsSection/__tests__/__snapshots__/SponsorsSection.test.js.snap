// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SponsorsSection should render with many props assigned 1`] = `
<div
  className="Container gray"
>
  <div
    className="content"
  >
    <div
      className="headingContainer"
    >
      <h3
        className="Heading underline"
      >
        <div
          className="hashLinkContainer"
          data-testid="Heading Content sponsors-link"
        >
          <a
            data-testid="Hash Link"
            href="#sponsors-link"
            id="sponsors-link"
          >
            <span
              className="ScreenReaderOnly"
            >
              Scroll Link for 
              Sponsors
            </span>
            <svg
              className="icon"
            />
          </a>
          Sponsors
        </div>
      </h3>
    </div>
    <div
      className="headingContainer"
    >
      <h4
        className="Heading"
      >
        <div
          className="hashLinkContainer"
          data-testid="Heading Content paid-sponsors-link"
        >
          <a
            data-testid="Hash Link"
            href="#paid-sponsors-link"
            id="paid-sponsors-link"
          >
            <span
              className="ScreenReaderOnly"
            >
              Scroll Link for 
              Paid Sponsors
            </span>
            <svg
              className="icon"
            />
          </a>
          Paid Sponsors
        </div>
      </h4>
    </div>
    <p>
      These sponsors have donated cash to help keep the mission going here at Operation Code. They may or may not have also donated to our cause via others means.
    </p>
    <div
      className="columnsContainer"
    >
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://slack.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Slack logo"
              className="logo large"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/slack.png"
            />
          </noscript>
          <img
            alt="Slack logo"
            className="logo large"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/slack.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://threatstack.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Threat Stack logo"
              className="logo large"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/threat_stack.png"
            />
          </noscript>
          <img
            alt="Threat Stack logo"
            className="logo large"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/threat_stack.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://usbank.com/index.html"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="US Bank logo"
              className="logo large"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/usbank.png"
            />
          </noscript>
          <img
            alt="US Bank logo"
            className="logo large"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/usbank.png"
          />
        </a>
      </div>
    </div>
    <div
      className="headingContainer"
    >
      <h4
        className="Heading"
      >
        <div
          className="hashLinkContainer"
          data-testid="Heading Content in-kind-sponsors-link"
        >
          <a
            data-testid="Hash Link"
            href="#in-kind-sponsors-link"
            id="in-kind-sponsors-link"
          >
            <span
              className="ScreenReaderOnly"
            >
              Scroll Link for 
              In-Kind Sponsors
            </span>
            <svg
              className="icon"
            />
          </a>
          In-Kind Sponsors
        </div>
      </h4>
    </div>
    <p>
      The following organizations have helped Operation Code through services, products, advertisements, scholarships, or sponsorships. We thank them for their contributions.
    </p>
    <div
      className="columnsContainer"
    >
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://1password.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="1Password logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/1password.png"
            />
          </noscript>
          <img
            alt="1Password logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/1password.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://apexsystems.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="APEX Systems logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/apex_systems.png"
            />
          </noscript>
          <img
            alt="APEX Systems logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/apex_systems.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://github.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="GitHub logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/github.png"
            />
          </noscript>
          <img
            alt="GitHub logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/github.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://hackerrank.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="HackerRank logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/hackerrank.png"
            />
          </noscript>
          <img
            alt="HackerRank logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/hackerrank.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://heroku.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Heroku logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/heroku.png"
            />
          </noscript>
          <img
            alt="Heroku logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/heroku.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://logrocket.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="LogRocket logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/logrocket.png"
            />
          </noscript>
          <img
            alt="LogRocket logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/logrocket.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://oreilly.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="O'Reilly Media logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/oreilly.png"
            />
          </noscript>
          <img
            alt="O'Reilly Media logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/oreilly.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://zapier.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Zapier logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/zapier.png"
            />
          </noscript>
          <img
            alt="Zapier logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/zapier.png"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`SponsorsSection should render with required props 1`] = `
<div
  className="Container gray"
>
  <div
    className="content"
  >
    <div
      className="headingContainer"
    >
      <h3
        className="Heading underline"
      >
        <div
          className="hashLinkContainer"
          data-testid="Heading Content sponsors-link"
        >
          <a
            data-testid="Hash Link"
            href="#sponsors-link"
            id="sponsors-link"
          >
            <span
              className="ScreenReaderOnly"
            >
              Scroll Link for 
              Sponsors
            </span>
            <svg
              className="icon"
            />
          </a>
          Sponsors
        </div>
      </h3>
    </div>
    <div
      className="headingContainer"
    >
      <h4
        className="Heading"
      >
        <div
          className="hashLinkContainer"
          data-testid="Heading Content paid-sponsors-link"
        >
          <a
            data-testid="Hash Link"
            href="#paid-sponsors-link"
            id="paid-sponsors-link"
          >
            <span
              className="ScreenReaderOnly"
            >
              Scroll Link for 
              Paid Sponsors
            </span>
            <svg
              className="icon"
            />
          </a>
          Paid Sponsors
        </div>
      </h4>
    </div>
    <p>
      These sponsors have donated cash to help keep the mission going here at Operation Code. They may or may not have also donated to our cause via others means.
    </p>
    <div
      className="columnsContainer"
    >
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://slack.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Slack logo"
              className="logo large"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/slack.png"
            />
          </noscript>
          <img
            alt="Slack logo"
            className="logo large"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/slack.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://threatstack.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Threat Stack logo"
              className="logo large"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/threat_stack.png"
            />
          </noscript>
          <img
            alt="Threat Stack logo"
            className="logo large"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/threat_stack.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://usbank.com/index.html"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="US Bank logo"
              className="logo large"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/usbank.png"
            />
          </noscript>
          <img
            alt="US Bank logo"
            className="logo large"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/usbank.png"
          />
        </a>
      </div>
    </div>
    <div
      className="headingContainer"
    >
      <h4
        className="Heading"
      >
        <div
          className="hashLinkContainer"
          data-testid="Heading Content in-kind-sponsors-link"
        >
          <a
            data-testid="Hash Link"
            href="#in-kind-sponsors-link"
            id="in-kind-sponsors-link"
          >
            <span
              className="ScreenReaderOnly"
            >
              Scroll Link for 
              In-Kind Sponsors
            </span>
            <svg
              className="icon"
            />
          </a>
          In-Kind Sponsors
        </div>
      </h4>
    </div>
    <p>
      The following organizations have helped Operation Code through services, products, advertisements, scholarships, or sponsorships. We thank them for their contributions.
    </p>
    <div
      className="columnsContainer"
    >
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://1password.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="1Password logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/1password.png"
            />
          </noscript>
          <img
            alt="1Password logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/1password.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://apexsystems.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="APEX Systems logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/apex_systems.png"
            />
          </noscript>
          <img
            alt="APEX Systems logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/apex_systems.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://github.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="GitHub logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/github.png"
            />
          </noscript>
          <img
            alt="GitHub logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/github.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://hackerrank.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="HackerRank logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/hackerrank.png"
            />
          </noscript>
          <img
            alt="HackerRank logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/hackerrank.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://heroku.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Heroku logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/heroku.png"
            />
          </noscript>
          <img
            alt="Heroku logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/heroku.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://logrocket.com/"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="LogRocket logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/logrocket.png"
            />
          </noscript>
          <img
            alt="LogRocket logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/logrocket.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://oreilly.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="O'Reilly Media logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/oreilly.png"
            />
          </noscript>
          <img
            alt="O'Reilly Media logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/oreilly.png"
          />
        </a>
      </div>
      <div
        className="PartnerLogoLink"
      >
        <a
          href="https://zapier.com"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="ScreenReaderOnly"
          >
            Opens in new window
          </span>
          <noscript>
            <img
              alt="Zapier logo"
              className="logo"
              src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/zapier.png"
            />
          </noscript>
          <img
            alt="Zapier logo"
            className="logo"
            data-src="https://operation-code-assets.s3.us-east-2.amazonaws.com/partnerLogos/zapier.png"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
